### PR TITLE
adding types for translations

### PR DESCRIPTION
**Pull request by virdesai**
# Submit a pull request

## CLA

- [ ] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required).
- [ ] The code changes follow best practices
- [ ] Code changes are tested (add some information if not applicable)

### CODE REVIEW

**Review by nhannah — APPROVED** [review]

**Review by DanC5 — APPROVED** [review]

File: src/types.ts [hunk 1]
```diff
@@ -1656,3 +1672,61 @@ export type User<UserType = UnknownType> = UserType & {
 };
 
 export type TypingStartEvent = Event;
+
+// should match the supportedLanguages here https://github.com/GetStream/chat/blob/master/translation/language.go
```
Comment (ferhatelmas) [inline src/types.ts:1676]: can we remove this ?

Comment (virdesai) [reply src/types.ts:1676]: sure

**Review by ferhatelmas — APPROVED** [review]
LGTM

File: src/types.ts (hunk 1 above)
Comment (ferhatelmas) [reply src/types.ts]: thanks

**Pull request merged**